tests/testthat/test-add_p.tbl_summary.R

```diff
@@ -1,6 +1,78 @@
 skip_on_cran()
 skip_if_not(is_pkg_installed(c("broom", "lme4", "broom.helpers"), ref = "cardx"))
 
+test_that("Regression test for system-dependent test for pvalue", {
```

Comment:
@ddsjoberg This is what you suggested and it works for older versions and new versions